packages/core/src/components/widgets/ColorWidget.js

```diff
@@ -9,7 +9,43 @@ function ColorWidget(props) {
       widgets: { BaseInput },
```

Comment:
This is now called `BaseInputTemplate` and is obtained via a call like this:
```
  const { options, registry } = props;
  const BaseInputTemplate = getTemplate<"BaseInputTemplate", T, F>(
    "BaseInputTemplate",
    registry,
    options
  );
```

Reply:
Hi @heath-freenome 
I reverted all of the package*.json and updated the code for the new version. Thank you